railties/test/application/initializers/frameworks_test.rb

```diff
@@ -50,15 +50,15 @@ def teardown
       assert_equal "test.rails", ActionMailer::Base.default_url_options[:host]
     end
 
-    test "Default to HTTPS for ActionMailer URLs when force_ssl is on" do
+    test "sets URL builder default protocol to HTTPS when force_ssl is on" do
```

Comment:
The tests in this file are grouped by framework; could you move this one down to the Action Dispatch section that starts at line 172?

https://github.com/rails/rails/blob/a6711d6e9d72ee72615d5c24d761dff5e94b6573/railties/test/application/initializers/frameworks_test.rb#L172